README.md

```diff
@@ -61,7 +59,7 @@ Current official backends:
 Current official terminals:
 
 - [ancypwn-terminal-alacritty](https://github.com/Escapingbug/ancypwn-terminal-alacritty)
-- [ancypwn-terminal-iterm2](https://github.com/Escapingbug/ancypwn-terminal-iterm2)
+- [ancypwn-terminal-iterm2](https://github.com/shizhongpwn/ancypwn-terminal-iterm2.git)
```

Comment:
What is the problem with the current solution? If you please, you might fire an PR to that repo as well instead of just replacing the "official" one?